semgrep/dependencyparser/models.py

```diff
@@ -19,7 +19,7 @@ class PackageManagers(Enum):
     PYPI = "pypy"
 
 
-@attr.s(auto_attribs=True, frozen=True, eq=True, order=True)
+@define(auto_attribs=True, frozen=True, eq=True, order=True)
```

Comment:
why not using simply @dataclass? Isn't it equivalent to this long @define?

Reply:
here: https://www.attrs.org/en/stable/why.html#data-classes

Reply:
primarily

> There is a long list of features that were sacrificed for the sake of simplicity and while the most obvious ones are validators, converters, [equality customization](https://www.attrs.org/en/stable/comparison.html#custom-comparison), or [extensibility](https://www.attrs.org/en/stable/extending.html) in general